devtools-integration-tests/src/test/resources/debug/single-file-tests/hello_service.bal

```diff
@@ -17,26 +17,21 @@
 import ballerina/http;
 import ballerina/test;
 
-service hello on new http:Listener(9090) {
-    resource function sayHello(http:Caller caller,
+service on new http:Listener(9090) {
+    resource function get sayHello(http:Caller caller,
```

Comment:
Can we merge these two lines (L21 and L22)?

Reply:
Fixed